jsconfig.json

```diff
@@ -1,20 +1,16 @@
 {
   "compilerOptions": {
-    "target": "ESNext",
+    "target": "esnext",
     "module": "NodeNext",
     "moduleResolution": "NodeNext",
-    "jsx": "preserve",
-    "declaration": true,
     "allowJs": true,
+    "checkJs": true,
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
-    "strict": true,
+    "strict": false,
     "skipLibCheck": true,
-    "noImplicitAny": false,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "strictNullChecks": false,
-    "useUnknownInCatchVariables": false
+    "resolveJsonModule": true,
+
```

Comment:
empty line? Why this changes?

Reply:
This is the config file for VSCode and it only used by it. The empty line is not needed in theory, while I just summarized the other changes by settings `strict=false` and removing overrides.